[CI] RateTests testGroupingAggregate {TestCase=<0 doubles>} failing

**Build Scans:**
- [elasticsearch-periodic #10665 / openjdk21_checkpart3_java-matrix](https://gradle-enterprise.elastic.co/s/l66532by4rkc4)
- [elasticsearch-pull-request #92629 / part-3](https://gradle-enterprise.elastic.co/s/etmwgzf7vqkae)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:esql:test" --tests "org.elasticsearch.xpack.esql.expression.function.aggregate.RateTests.testGroupingAggregate {TestCase=<-0 doubles>}" -Dtests.seed=7E37300283E80B57 -Dtests.locale=nb-SJ -Dtests.timezone=Europe/Guernsey -Druntime.java=21
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.esql.expression.function.aggregate.RateTests),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('testGroupingAggregate%20%7BTestCase%3D%3C0%20doubles%3E%7D'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: 
Expected: (a value equal to or greater than <-0.0> and a value less than or equal to <0.0>)
     but: a value less than or equal to <0.0> <1.3679245091320105E-14> was greater than <0.0>
```

**Issue Reasons:**
- [main] 2 failures in test testGroupingAggregate {TestCase=<0 doubles>} (0.7% fail rate in 297 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -564,28 +564,12 @@ tests:
 - class: org.elasticsearch.cluster.metadata.MetadataCreateIndexServiceTests
   method: testIndexSettingProviderPrivateSetting
   issue: https://github.com/elastic/elasticsearch/issues/134846
-- class: org.elasticsearch.xpack.esql.expression.function.aggregate.RateTests
-  method: testGroupingAggregate {TestCase=<-0 doubles>}
-  issue: https://github.com/elastic/elasticsearch/issues/134847
-- class: org.elasticsearch.xpack.esql.expression.function.aggregate.RateTests
-  method: testGroupingAggregate {TestCase=<0 doubles>}
-  issue: https://github.com/elastic/elasticsearch/issues/134849
 - class: org.elasticsearch.repositories.blobstore.testkit.analyze.MinioRepositoryAnalysisRestIT
   method: testRepositoryAnalysis
   issue: https://github.com/elastic/elasticsearch/issues/134853
 - class: org.elasticsearch.xpack.esql.action.CrossClusterAsyncQueryStopIT
   method: testStopQueryInlinestats
   issue: https://github.com/elastic/elasticsearch/issues/134854
-- class: org.elasticsearch.xpack.esql.expression.function.aggregate.RateTests
-  method: testGroupingAggregate {TestCase=<1000000000 ints>}
-  issue: https://github.com/elastic/elasticsearch/issues/134855
-- class: org.elasticsearch.xpack.esql.expression.function.aggregate.RateTests
-  method: testGroupingAggregate {TestCase=<positive longs>}
-  issue: https://github.com/elastic/elasticsearch/issues/134856
-- class: org.elasticsearch.xpack.esql.expression.function.aggregate.RateTests
-  method: testGroupingAggregate {TestCase=<1.0E9 doubles>}
-  issue: https://github.com/elastic/elasticsearch/issues/134860
-
 # Examples:
 #
 #  Mute a single test case in a YAML test suite:

